src/test/resources/sftp/local/OutBound/invalid-json.json

```diff
@@ -0,0 +1 @@
+- Invalid : JSON
```

Comment:
I think the filename should still be as per fileNaming pattern of  `status-d17.json` . The content seems ok for an invalid json

Reply:
I've deleted this file because the json processing wasn't required as part of this ticket.